### PR TITLE
Added CircleCI config and kube deployment script

### PR DESCRIPTION
**Pull request by DanielBrierton**

### CODE REVIEW

File: .circleci/kube.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  HOST=$PROD_HOST
+  echo "$PROD_CA_CERT" | base64 -i --decode > ca.pem
+  echo "$PROD_ADMIN_KEY" | base64 -i --decode > admin-key.pem
+  echo "$PROD_ADMIN_CERT" | base64 -i --decode > admin.pem
+else
+  exit 0
+fi
+docker build --rm=false -t coderdojo/events-service:"${GIT_SHA1}-${TIMESTAMP}" .
+docker login -u "$DOCKER_USER" -p "$DOCKER_PASS" -e "$DOCKER_EMAIL"
+docker push coderdojo/events-service:"${GIT_SHA1}-${TIMESTAMP}"
```
Comment (Wardormeur) [inline .circleci/kube.sh]: Do we have to create a repo before pushing ?

Comment (DanielBrierton) [reply .circleci/kube.sh]: Good point. I'd imagine we do.

Comment (DanielBrierton) [reply .circleci/kube.sh]: Done https://hub.docker.com/r/coderdojo/events-service/

File: package.json [hunk 2]
```diff
@@ -11,7 +11,10 @@
     "start": "node index.js",
     "dev": "nodemon index.js",
     "lint": "eslint .",
-    "precommit": "lint-staged"
+    "precommit": "lint-staged",
+    "unit": "HOSTNAME=http://platform.local mocha -t 3000 --require ./test/helper.js ./test/unit --recursive --reporter=nyan --forbid-pending",
+    "int": "NODE_ENV=test mocha -t 10000 --require ./test/helper.js ./test/integration --recursive --reporter=nyan --forbid-pending",
+    "test": "npm run unit && npm run int"
```
Comment (Wardormeur) [inline package.json]: goto yarn

**Pull request merged**